test/dist_autograd_test.py

```diff
@@ -1135,7 +1135,9 @@ def test_backward_node_failure(self):
                     if rank % 2 != 0:
                         wait_until_node_failure(rank)
 
-                with self.assertRaisesRegex(RuntimeError, "Request aborted during client shutdown"):
+                with self.assertRaisesRegex(RuntimeError,
```

Comment:
Can you add a comment that explains what are the two possible errors here and why both are ok?